mappings/net/minecraft/item/Item.mapping

```diff
@@ -1,149 +1,182 @@
-			ARG 1 itemGroup
+			ARG 1 settings
+		METHOD a group (Lbas;)Lbbw$a;
+			ARG 1 group
 		METHOD a recipeRemainder (Lbbw;)Lbbw$a;
 			ARG 1 recipeRemainder
 		METHOD a rarity (Lbco;)Lbbw$a;
 			ARG 1 rarity
-		METHOD b durabilityIfNotSet (I)Lbbw$a;
-			ARG 1 durability
-		METHOD c durability (I)Lbbw$a;
-			ARG 1 durability
-	FIELD a GETTER_DAMAGED Lbca;
-	FIELD b GETTER_DAMAGE Lbca;
-	FIELD c GETTER_HAND Lbca;
-	FIELD d GETTER_COOLDOWN Lbca;
-	FIELD e GETTER_CUSTOM_MODEL_DATA Lbca;
-	FIELD f BLOCK_ITEM_MAP Ljava/util/Map;
-	FIELD g MODIFIER_DAMAGE Ljava/util/UUID;
-	FIELD h MODIFIER_SWING_SPEED Ljava/util/UUID;
-	FIELD i random Ljava/util/Random;
-	FIELD j itemGroup Lbas;
-	FIELD k PROPERTIES Ljava/util/Map;
+		METHOD b maxDamageIfAbsent (I)Lbbw$a;
+			ARG 1 maxDamage
+		METHOD c maxDamage (I)Lbbw$a;
+			ARG 1 maxDamage
+	FIELD a DAMAGED_PROPERTY_GETTER Lbca;
+	FIELD b DAMAGE_PROPERTY_GETTER Lbca;
+	FIELD c LEFTHANDED_PROPERTY_GETTER Lbca;
+	FIELD d COOLDOWN_PROPERTY_GETTER Lbca;
+	FIELD e CUSTOM_DATA_PROPERTY_GETTER Lbca;
+	FIELD f BLOCK_ITEMS Ljava/util/Map;
+	FIELD g ATTACK_DAMAGE_MODIFIER_UUID Ljava/util/UUID;
+	FIELD h ATTACK_SPEED_MODIFIER_UUID Ljava/util/UUID;
+	FIELD i RANDOM Ljava/util/Random;
+	FIELD j group Lbas;
+	FIELD k propertyGetters Ljava/util/Map;
 	FIELD l rarity Lbco;
-	FIELD m fullStackSize I
-	FIELD n durability I
+	FIELD m maxCount I
+	FIELD n maxDamage I
 	FIELD o recipeRemainder Lbbw;
 	FIELD p translationKey Ljava/lang/String;
-	FIELD q foodSetting Laxz;
+	FIELD q foodSettings Laxz;
 	METHOD <init> (Lbbw$a;)V
 		ARG 1 settings
-	METHOD P_ isMap ()Z
+	METHOD P_ isComplex ()Z
 	METHOD a getTranslationKey ()Ljava/lang/String;
-	METHOD a getAttributeModifiers (Lain;)Lcom/google/common/collect/Multimap;
-		ARG 1 equiptmentSlot
-	METHOD a isInItemGroup (Lbas;)Z
-	METHOD a appendItemsForGroup (Lbas;Lfj;)V
-	METHOD a getRawIdByItem (Lbbw;)I
+	METHOD a getModifiers (Lain;)Lcom/google/common/collect/Multimap;
+		ARG 1 slot
+	METHOD a isIn (Lbas;)Z
+		ARG 1 group
+	METHOD a appendStacks (Lbas;Lfj;)V
+		ARG 1 group
+		ARG 2 stacks
+	METHOD a getRawId (Lbbw;)I
 		ARG 0 item
-	METHOD a onEntityDamaged (Lbcb;Lair;Lair;)Z
+	METHOD a attack (Lbcb;Lair;Lair;)Z
 		ARG 1 stack
 		ARG 2 target
-		ARG 3 livingEntity
-	METHOD a interactWithEntity (Lbcb;Lavy;Lair;Lahc;)Z
+		ARG 3 attacker
+	METHOD a useOnEntity (Lbcb;Lavy;Lair;Lahc;)Z
 		ARG 1 stack
-		ARG 2 player
-		ARG 3 target
+		ARG 2 user
+		ARG 3 entity
+		ARG 4 hand
 	METHOD a canRepair (Lbcb;Lbcb;)Z
-		ARG 1 tool
+		ARG 1 stack
 		ARG 2 ingredient
-	METHOD a onEntityTick (Lbcb;Lbhi;Laii;IZ)V
+	METHOD a tick (Lbcb;Lbhi;Laii;IZ)V
 		ARG 1 stack
 		ARG 2 world
 		ARG 3 entity
-		ARG 4 invSlot
+		ARG 4 slot
 		ARG 5 selected
-	METHOD a onItemFinishedUsing (Lbcb;Lbhi;Lair;)Lbcb;
+	METHOD a finishUsing (Lbcb;Lbhi;Lair;)Lbcb;
 		ARG 1 stack
 		ARG 2 world
-		ARG 3 livingEntity
-	METHOD a onItemStopUsing (Lbcb;Lbhi;Lair;I)V
+		ARG 3 user
+	METHOD a onStoppedUsing (Lbcb;Lbhi;Lair;I)V
```

Comment:
Should this also drop the `on` prefix?

Reply:
I retained `on` prefix for event-based methods as it is the current standard in Yarn afaik. Would rather that change be a global migration rather than create inconsistencies now I think.

Reply:
If you are referring to the differing name pattern for `finishUsing`, it is because that is not an event listener, it is a transformer that returns a "finished" stack. See `FoxEntity#tickMovement` when the fox eats food, for example.

Reply:
Ah, yeah, had thought the prefix was being dropped in other places. 👍